<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClubForms</class>
 <widget class="QWidget" name="ClubForms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mi Deporte Movil</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/img/ico2.ico</normaloff>:/img/ico2.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(57, 57, 57, 255), stop:1 rgba(255, 255, 255, 255))</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_12">
   <item row="1" column="0" rowspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0.7);

selection-color: white;  
selection-background-color:rgba(0,0,0,0.5);     

border-radius: 10px;</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Reservas</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0" colspan="2">
          <widget class="QGroupBox" name="groupBox_10">
           <property name="title">
            <string>Canchas</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QComboBox" name="cmbGameFields">
              <property name="styleSheet">
               <string notr="true">color:white;
border-color: grey;</string>
              </property>
              <property name="currentText">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCalendarWidget" name="calendarWidget">
              <property name="styleSheet">
               <string notr="true">QCalendarWidget QAbstractItemView
{
background-color: rgb(192,192,192); /* цвет фона текущего месяца */
selection-background-color: black; /* цвет фона выбранного дня */
selection-color: white; /* цвет текста выбранного дня */
}
QCalendarWidget QMenu {
  	color: white;
  	background-color: rgb(100, 100, 100);
	selection-background-color: black; /* цвет фона выбранного дня */
	selection-color: white; /* цвет текста выбранного дня */
  }

QCalendarWidget QMenu::hover{
	color: rgb(158, 158, 158);
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>Reservas del dia</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <widget class="QListWidget" name="listWidget">
              <property name="styleSheet">
               <string notr="true">color:white;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Anuncios</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QTextEdit" name="textEdit">
           <property name="styleSheet">
            <string notr="true">color:white;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Crear Anuncio</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0" colspan="2">
             <widget class="QTextEdit" name="teMessage">
              <property name="styleSheet">
               <string notr="true">color:white;</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pbPublic">
              <property name="font">
               <font>
                <family>Adobe Myungjo Std M</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">#pbPublic {
	color: #333;
	border: 2px solid #555;
	border-radius: 11px;
	padding: 5px;
	background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #888);
	min-width: 80px;
}
#pbPublic:hover {
	background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #bbb);
}
#pbPublic:pressed {
	background: qradialgradient(cx: 0.4, cy: -0.1,
	fx: 0.4, fy: -0.1,
	radius: 1.35, stop: 0 #fff, stop: 1 #ddd);
}</string>
              </property>
              <property name="text">
               <string>Publicar</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pbClear">
              <property name="font">
               <font>
                <family>Adobe Myungjo Std M</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">#pbClear {
	color: #333;
	border: 2px solid #555;
	border-radius: 11px;
	padding: 5px;
	background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #888);
	min-width: 80px;
}
#pbClear:hover {
	background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #bbb);
}
#pbClear:pressed {
	background: qradialgradient(cx: 0.4, cy: -0.1,
	fx: 0.4, fy: -0.1,
	radius: 1.35, stop: 0 #fff, stop: 1 #ddd);
}</string>
              </property>
              <property name="text">
               <string>Borrar</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Productos</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QListWidget" name="productsList">
           <property name="styleSheet">
            <string notr="true">color:white;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="img.qrc">:/img/patrocinio/nike.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="img.qrc">:/img/patrocinio/penn.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="img.qrc">:/img/patrocinio/babolat.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="img.qrc">:/img/patrocinio/adidas_PNG8.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="img.qrc">:/img/patrocinio/Wilson-logo.svg.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
